movies/src/components/common/multiSelectInput/MultiSelectInput.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import MultiSelect from 'react-multi-select-component';
```

Comment:
did you forget tot add this module to dependencies?

Reply:
I forgot to commit my package.json, Now all dependencies are presented.